extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -240,17 +244,24 @@ public LDAPAuthenticatedUser authenticateUser(Credentials credentials)
 
         try {
 
+            LdapConnectionConfig ldapConnectionConfig =
+                    ((LdapNetworkConnection) ldapConnection).getConfig();
```

Comment:
I'll see what makes the most sense - maybe just converting everything to LdapNetworkConnection  is the most straight-forward, since that's really the only implementation that ever gets used.

Reply:
Nevermind...I was able to clean this stuff up.